distribution/docs/src/main/resources/templates/build/appendices.ftl

```diff
@@ -0,0 +1,20 @@
+<#list appendixIntros?sort_by("order") as ai>
```

Comment:
these template files are not new, just relocated. Somehow git lost track of them.